src/test/HelmetTest.jsx

```diff
@@ -628,5 +629,37 @@ describe("Helmet", () => {
 
             Helmet.canUseDOM = true;
         });
+
+        it("will not update the DOM if updated props are unchanged", (done) => {
+            const old = HelmetUnwrapped.onDOMChange;
+            let changesToDOM = 0;
+            HelmetUnwrapped.onDOMChange = (state) => {
+                changesToDOM++;
+                return state;
+            };
+
+            React.render(
+                <Helmet
+                    title={"Test Title"}
+                    meta={[{"name": "description", "content": "Test description"}]}
+                />,
+                container
+            );
+
+            // Re-rendering will pass new props to an already mounted Helmet
+            React.render(
+                <Helmet
+                    title={"Test Title"}
+                    meta={[{"name": "description", "content": "Test description"}]}
+                />,
+                container
+            );
+
+            setTimeout(() => {
+                expect(changesToDOM).to.equal(1);
+                HelmetUnwrapped.onDOMChange = old;
+                done();
+            }, 1000);
+        });
```

Comment:
Yes I'm looking for verification that it only renders Helmet one time in a scenario with two nested components.

The test above would cover that wouldn't it?